.changeset/sharp-kiwis-eat.md

```diff
@@ -0,0 +1,9 @@
+---
+'@commercetools-test-data/standalone-price': major
```

Comment:
@tdeekens
Thanks for pointing it out. I must have followed the prompts too fast after a `pnpm changeset` and thought it was set as major automatically.
There was a slight uncertainty that it could affect other tests (https://github.com/commercetools/test-data/pull/556#discussion_r1578256475) but considering there are no potential breaking changes it's more of a minor bump than a major